php/index.php

```diff
@@ -0,0 +1,18 @@
+<?php
+
+use Ionic\CustomAuthentication;
+
+require 'vendor/autoload.php';
+
+Flight::route('/auth',function () {
+    try {
+        $redirect_uri = CustomAuthentication::process();
+        Flight::redirect($redirect_uri);
+    } catch (\Exception $e) {
+        Flight::json(['error' => $e->getMessage(), 'code' => $e->getCode()], 401);
+    }
+});
+
+Flight::start();
+
+?>
```

Comment:
Just a style issue, but am I wrong in saying that the trailing tag is generally left out in modern PHP?